### ISSUE TITLE
After upgrading to Anki 2.1.50, media are created in incorrect folder

### ISSUE THREAD

**Issue by mhujer** [sviatoslav-lebediev/anki-quizlet-importer-extended#32]
I'm using Anki with a specific data folder (launching Anki with `C:\Anki\AnkiApp\anki.exe -b c:\Anki\AnkiData\`).

Since upgrading to Anki 2.1.50 (the issue persists in 2.1.51), media downloaded from Quizlet are created in the application directory (`C:\Anki\AnkiApp`) and not in data directory (`C:\Anki\AnkiData\Martin\collection.media\`).

I was wondering if anyone else noticed this? Do you think it is related to the plugin or is it a bug in the Anki?

(The workaround is simple: after downloading new set, I just move the media files to the correct location)

Small set for testing: https://quizlet.com/688467316/test-flash-cards/

**Comment by mhujer** [comment]
I just had an idea which may help with debugging the issue: I tried to drag&drop an image to a new card and the file was created in the correct directory.

**Comment by sviatoslav-lebediev** [comment]
ok, I've found the place where we can fix this "issue" :) the only problem is that I can't find an Anki API that provides a method for getting a base folder :(  here is the only code that I've found https://github.com/ankitects/anki/blob/e6d990a44e2f1d3ef9dfb1303c6049f749dd01f0/qt/aqt/__init__.py#L351
I will try to find a way how to fix it but so far you can go and manually change this line  https://github.com/sviatoslav-lebediev/anki-quizlet-importer-extended/blob/89d62a112d3f496cdc5f0bb8db8048e4077ac0a3/__init__.py#L384 on your machine
```diff
- file_name = "quizlet-" + suffix if suffix else  "quizlet-" + url.split('/')[-1]
+ file_name = "/Users/username/data_folder/quizlet-" + suffix if suffix else  "quizlet-" + url.split('/')[-1]
```
p.s
that's how you can find a file 
![image](https://user-images.githubusercontent.com/19693768/167255939-e32ab860-ab4a-4d4f-8e54-4b23775b6319.png)

**Comment by mhujer** [comment]
@sviatoslav-lebediev Hello, thanks for the hint! I did some digging around anki repo and found that there is "media manager" with the [`dir()` method](https://github.com/ankitects/anki/blob/e6d990a44e2f1d3ef9dfb1303c6049f749dd01f0/pylib/anki/media.py#L61) and conveniently, the media manager is available on the collection, so in the end, it was an easy fix (see #33).

```diff
diff --git a/__init__.py b/__init__.py
index 0b88c66..bca5ec6 100644
--- a/__init__.py
+++ b/__init__.py
@@ -385,7 +385,7 @@ def fileDownloader(self, url, suffix=''):
         # get original, non-mobile version of images
         r = requests.get(url, stream=True, verify=False, headers=headers)
         if r.status_code == 200:
-            with open(file_name, 'wb') as f:
+            with open(mw.col.media.dir() + "/" + file_name, 'wb') as f:
                 r.raw.decode_content = True
                 shutil.copyfileobj(r.raw, f)
         return file_name
```

**Comment by sviatoslav-lebediev** [comment]
cool, thx for a PR. I will test/merge it tomorrow.
